Shared/Samples/Set viewpoint rotation/SetViewpointRotationView.swift

```diff
@@ -39,9 +39,9 @@ struct SetViewpointRotationView: View {
                 }
                 .overlay(alignment: .topTrailing) {
                     Compass(
-                        viewpointRotation: $rotation,
-                        autoHide: false
+                        viewpointRotation: $rotation
                     )
+                    .autoHideDisabled()
                     .padding()
```

Comment:
We should probably single line the init since it only has 1 param now

```swift
                    Compass(viewpointRotation: $rotation)
                        .autoHideDisabled()
                        .padding()
```